enf/network_ratelimit.go

```diff
@@ -0,0 +1,63 @@
+package enf
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// NetworkRateLimits represents the values of rate limits for a network.
+type NetworkRateLimits struct {
+	PacketsPerSecond *int  `json:"packets_per_second"`
+	PacketsBurstSize *int  `json:"packets_burst_size"`
+	BytesPerSecond   *int  `json:"bytes_per_second"`
+	BytesBurstSize   *int  `json:"bytes_burst_size"`
+	Inherit          *bool `json:"inherit"`
+}
+
+// standardRateLimitResponse represents the standard API response for
+// all relevant network rate limit endpoints.
+type networkRateLimitResponse struct {
+	Data []*NetworkRateLimits   `json:"data"`
+	Page map[string]interface{} `json:"page"`
+}
+
+// GetDefaultRateLimits gets the default rate limits for the given network address.
+func (s *NetworkService) GetDefaultRateLimits(ctx context.Context, network string) (*NetworkRateLimits, *http.Response, error) {
+	endpoint := fmt.Sprintf("api/xcr/v2/nws/%v/ep_rate_limits/default", network)
+	body, resp, err := s.client.get(ctx, endpoint, new(networkRateLimitResponse))
+	if err != nil {
+		return nil, resp, err
+	}
+	return body.(*networkRateLimitResponse).Data[0], resp, nil
+}
+
+// GetMaxRateLimits gets the max rate limits for the given network address.
+func (s *NetworkService) GetMaxRateLimits(ctx context.Context, network string) (*NetworkRateLimits, *http.Response, error) {
+	endpoint := fmt.Sprintf("api/xcr/v2/nws/%v/ep_rate_limits/max", network)
+	body, resp, err := s.client.get(ctx, endpoint, new(networkRateLimitResponse))
+	if err != nil {
+		return nil, resp, err
+	}
+	return body.(*networkRateLimitResponse).Data[0], resp, nil
+}
+
+// SetDefaultRateLimits sets the default rate limits for the given network address.
+func (s *NetworkService) SetDefaultRateLimits(ctx context.Context, values *NetworkRateLimits, network string) (*NetworkRateLimits, *http.Response, error) {
+	endpoint := fmt.Sprintf("api/xcr/v2/nws/%v/ep_rate_limits/default", network)
```

Comment:
This applies to all the service methods:

`endpoint` is confusing, since that is also an object type in our API.  `path` would be as accurate and not as confusing.

Reply:
Okay, I can change the naming for this.